### PR TITLE
Add static Builds page so users can rerun a build

### PR DESCRIPTION
**Pull request by smharley**
I didn't commit the changes to the `routes` that would be needed to view this page, figured we'd want to keep it hidden

* Refactor font-weight
* Refactor transitions
* Add spacing variables
* Build Builds page with static content

https://trello.com/c/FLrhuBGo

### CODE REVIEW

File: app/assets/stylesheets/pages/_builds.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
+.build-button {
+  background-color: $base-background-color;
+  border: 2px solid $base-border-color;
+  border-radius: 0 0 $base-border-radius $base-border-radius;
```
Comment (smharley) [inline app/assets/stylesheets/pages/_builds.scss]: Hmm I'm not sure `border-radius` shorthands work like that Hound.. I'll 👀 into it

**Comment by smharley** [comment]
image

File: app/assets/stylesheets/pages/_builds.scss (hunk 1 above)
Comment (smharley) [inline app/assets/stylesheets/pages/_builds.scss]: Are we using the default thoughtbot SCSS style? Shorthands of length `4` should be allowed:

https://github.com/thoughtbot/guides/blob/master/style/sass/.scss-lint.yml#L169

```
  Shorthand:
    enabled: true
    allowed_shorthands: [1, 2, 3, 4]
```

Comment (thorncp) [inline app/assets/stylesheets/pages/_builds.scss]: We aren't using the file you linked yet. There's a Hound copy at https://github.com/houndci/linters/blob/master/config/scss.yml that has probably gotten out of sync.

Comment (smharley) [inline app/assets/stylesheets/pages/_builds.scss]: Ah, looks like that's the case:

https://github.com/houndci/linters/blob/master/config/scss.yml#L169

```
  Shorthand:
    enabled: true
    allowed_shorthands: [1, 2, 3]
```

**Comment by thorncp** [comment]
Looks good to me! :mount_fuji:

**Pull request merged**